packages/core/i18n/core-i18n-browser-internal/src/i18n_eui_mapping.tsx

```diff
@@ -725,9 +725,12 @@ export const getEuiContextMapping = (): EuiTokensObject => {
     'euiFieldPassword.maskPassword': i18n.translate('core.euiFieldPassword.maskPassword', {
       defaultMessage: 'Mask password',
     }),
-    'euiFilePicker.clearSelectedFiles': i18n.translate('core.euiFilePicker.clearSelectedFiles', {
-      defaultMessage: 'Clear selected files',
-    }),
+    'euiFilePicker.removeSelectedAriaLabel': i18n.translate(
+      'core.euiFilePicker.removeSelectedAriaLabel',
+      {
+        defaultMessage: 'Remove selected files',
```

Comment:
Why was this copy changed from `Clear` to `Remove`? I don't think this functionality deletes anything.

Reply:
That change comes from https://github.com/elastic/eui/pull/7860 and originated in https://github.com/elastic/observability-dev/issues/3641.

The `Remove` EuiFilePicker button previously had an ARIA label, `Clear selected files`, that didn't match the button's displayed label. Now, the ARIA label matches the display label better. I agree that the usage of `remove` here could sometimes be misleading, but considering this button is only visible when you select a file from disk, it does make sense.